packages/algorand-js/test/integration/crowdfunding.ts

```diff
@@ -64,7 +63,7 @@ describe("Crowdfunding basic tests", function () {
     const appId = await runtime.addApp({ ...validFlags, appArgs: appArgs }, {}, program);
     const getGlobal = (key: string):
       bigint | Uint8Array |undefined => runtime.getGlobalState(appId, stringToBytes(key));
-    const johnPk = decodeAddress(john.address).publicKey;
+    const johnPk = addressToBytes(john.address);
```

Comment:
how about renaming `addressToBytes` -> `addressToPk`? BTW, I thought that address doesn't have a reverse function to Public Key.

Reply:
for int and string, we are using toBytes, and according to decodeAddress, it returns pk as Uint8Array. so looking from user perspective i think `addressToBytes` is ok(i added doc string addressToBytes about Public key).
should we rename it to `addressToPk` ?

Reply:
updated to `addressToPk`